README.md

```diff
@@ -85,6 +85,25 @@ Once the door unlocked, the activity opened by the SDK will close automatically
 
 ```
 
+#### Unlock a door by UUID
+The SDK has a method that will open an activity to pass the information once obtained the UUID beforehand.
+- You need the `uuid (String)`
```

Comment:
because 100% of the tiles will come as a tile id from a QR, which exports a String.
You will need to provision the SDK with a `String`, and I'll convert it to an `UUID` eventually (see video with the error case)

Reply:
I'm not really sure that's a good argument, everything is a string before its a UUID unless its a brand new UUID!